### ISSUE TITLE
Set up a published docs site

### ISSUE THREAD

**Issue by markerikson** [reduxjs/redux-starter-kit#48]
I'd like to set up an actual published docs site using Docusaurus.

First step would be to copy over the Docusaurus config that we're using with React-Redux:  https://github.com/reduxjs/react-redux/tree/master/website .

I can take care of setting up hosting and a domain name.  

From there, we can extract the docs content from our current README and move those into separate pages.

Please leave a comment if you'd like to work on this!

**Comment by markerikson** [comment]
I've got a Netlify site up at https://redux-starter-kit-docs.netlify.com that will host it.  Once the domain name is ready,  I'll just point it at that site.

In theory, this should mostly just need:

- Copy the existing `website` folder from React-Redux
- Rename all references from "react-redux" to "redux-starter-kit"
- Create a `docs` folder with some initial content
- Modify `website/sidebars.json` to point to the initial file

**Comment by sag1v** [comment]
Just to be clear... 
> Copy the existing website folder from React-Redux
> Create a docs folder with some initial content

You want to copy the website folder content to `redux-starter-kit/website`
and create the `docs` folder in `redux-starter-kit/website` right?

> Modify website/sidebars.json to point to the initial file

The initial file would be the existing `redux-starter-kit/website/build/redux-starter-kit/index.html` ?

**Comment by markerikson** [comment]
> You want to copy the website folder content to redux-starter-kit/website and create the docs folder in redux-starter-kit/website right?

Yeah, copy-pasting the site config from React-Redux would be the easiest way to get started setting it up for this repo.

The current `redux-starter-kit/website/build/redux-starter-kit/index.html` is a placeholder, just so I could say that the Netlify site has valid content when I filed the domain name request.  We'll remove that as soon as we have a Docusaurus build system set up.

**Comment by sag1v** [comment]
Cool, let me know if i need to change things. 
note that i copy the `index.html` from `build` to `docs`.

https://github.com/sag1v/redux-starter-kit

**Comment by markerikson** [comment]
Per my comment, the branch will need to have the config files changed to replace all React-Redux references with the starter kit's name and repo, and the site contents need to get changed.

**Comment by sag1v** [comment]
Do you see any other references I've missed?

**Comment by markerikson** [comment]
Yeah, several - left comments appropriately.
